reporter/utils.go

```diff
@@ -206,6 +259,16 @@ func (r *Reporter) matchAndSubmitCheckpoints(signer string) int {
 		// submit the checkpoint to Babylon
 		res, err := r.babylonClient.InsertBTCSpvProof(context.Background(), msgInsertBTCSpvProof)
 		if err != nil {
+			if errors.Is(err, btcctypes.ErrDuplicatedSubmission) {
+				r.logger.Infof("Checkpoint already submitted, for epoch: %d, tx1: %s, tx2: %s, height: %s",
+					ckpt.Epoch,
+					tx1Block.Txs[ckpt.Segments[0].TxIdx].Hash().String(),
+					tx2Block.Txs[ckpt.Segments[1].TxIdx].Hash().String(),
+					strconv.Itoa(int(tx1Block.Height)))
```

Comment:
This printing looks a bit weird how we are parsing the tx hash, but I see that is already being used in other parts of the code